examples/app/Embeddings.ipynb

```diff
@@ -20,36 +48,257 @@
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "500e19b1",
+   "id": "5eff5bca",
    "metadata": {},
    "outputs": [],
    "source": [
-    "data"
+    "clusterable_projection = umap.umap_project(\n",
+    "    embeddings, {\n",
+    "        'n_neighbors': 30,\n",
+    "        'min_dist': 0,\n",
+    "        'n_components': 2,\n",
+    "    }\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "583830d0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "clusters = hdbscan.hdbscan_cluster(clusterable_projection, {\n",
+    "            'min_samples': 10,\n",
+    "            'min_cluster_size': 60\n",
+    "        })"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "87c4ad19",
+   "metadata": {},
+   "source": [
+    "Then, we'll project the embeddings again for plotting. For efficient computation, weave isn't actually executing anything until weave call `weave.use` or `weave.show` on the built up computation (or \"compute graph\")."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "21f5917b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "projection = umap.umap_project(embeddings, {})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e3b7f76c",
+   "metadata": {},
+   "source": [
+    "## Combine data for plotting"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1d017836",
+   "metadata": {},
+   "source": [
+    "Finally, we'll combine our data into one big table so we can display the Pokemon name and types alongside our embedding. Here, we're calling `weave.use` on each weave object to execute the computation, and then we're merging them into a new list of dictionarys. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d642110d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "weave.show([{'x': x, 'y': y, 'k':k, 'd':d} for (x,y),k,d in \n",
+    "            zip(weave.use(projection),\n",
+    "                weave.use(clusters),\n",
+    "                weave.use(data))])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1cf9b267",
+   "metadata": {},
+   "source": [
+    "## Build and publish an interactive dashboard"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "daedd907",
+   "metadata": {},
+   "source": [
+    "Here are some Weave UI tips to build an interactive dashboard to get insights into your embeddings. You can add new panels by clicking the + button on the drawer that pops out on the right hand side of the cell. You can refer to other panels by their name, the first panel should be called `table`. Try converting the first panel to a `Plot` and then adding another `plot` panel that uses `table.selected_data` from the first panel. You can click the pencil to edit the configuration of your `plot`. "
+   ]
+  },
+  {
+   "attachments": {
+    "image.png": {
```

Comment:
this example you have is very cool! can you give the step-by-step instructions for the bar chart, or at least a bit more detail? also I think it's a good idea to share the url to the published version of this board—even these two panels configured, and even if the url is long—whatever you're willing to publish in your entity/weave as a public url. We might also consider making a specific wandb entity for publishing demo boards?

Reply:
Agreed with all of the above. Add more UI instructions and publish to new entity and make them public.